Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -153,16 +153,76 @@ final class TestDiscoveryCommand: CustomLLBuildCommand {
 
         stream <<< "import XCTest" <<< "\n\n"
 
+        stream <<< "@available(*, deprecated, message: \"Not actually deprecated. Marked as deprecated to allow inclusion of deprecated tests (which test deprecated functionality) without warnings\")" <<< "\n"
```

Comment:
Will this result in a deprecation warning whenever it's compiled?  Is there anyway around that, e.g. can the Swift compiler be asked to suppress the warning for this particular declaration?

Reply:
Never mind, I now see that this new code does exactly the same as the existing code below, so this seems fine.